plugin/core/transports.py

```diff
@@ -258,7 +260,8 @@ def start_subprocess() -> subprocess.Popen:
             start_subprocess
         )
     else:
-        process = start_subprocess()
+        if config.command:
+            process = start_subprocess()
```

Comment:
We should have some error handling and raise an error for the case when we expect `command` but it's not provided. I suppose we should only allow that when `tcp_port` is not set.